scripts/delete.py

```diff
@@ -19,7 +19,7 @@
 from cognite.client.data_classes.data_modeling import ViewId
 from cognite.client.data_classes.time_series import TimeSeries
 
-from .transformations_config import parse_transformation_configs
+# from scripts.transformations.transformations_config import parse_transformation_configs
```

Comment:
Just delete this. It's a leftover from the code that used the transformations-cli code (and delete.py was not refactored)